projects/nfdi-gdo-app/src/app/species-selector/species-selector.component.ts

```diff
@@ -92,6 +110,276 @@ export class SpeciesSelectorComponent implements OnInit, OnDestroy {
         'Sympetrum vulgatum',
     ];
 
+    readonly environmentLayers: EnvironmentLayer[] = [
+        {
+            id: '36574dc3-560a-4b09-9d22-d5945f2b8093',
+            name: 'NDVI',
+            colorizer: {
+                type: 'palette',
+                colors: {
+                    '0': [236, 224, 215, 0],
+                    '1': [235, 223, 214, 255],
+                    '2': [234, 222, 212, 255],
+                    '3': [234, 221, 211, 255],
+                    '4': [233, 221, 209, 255],
+                    '5': [232, 220, 208, 255],
+                    '6': [231, 219, 206, 255],
+                    '7': [231, 218, 205, 255],
+                    '8': [230, 217, 204, 255],
+                    '9': [229, 216, 202, 255],
+                    '10': [228, 215, 201, 255],
+                    '11': [227, 214, 199, 255],
+                    '12': [227, 214, 198, 255],
+                    '13': [226, 213, 197, 255],
+                    '14': [225, 212, 195, 255],
+                    '15': [224, 211, 194, 255],
+                    '16': [224, 210, 192, 255],
+                    '17': [223, 209, 191, 255],
+                    '18': [222, 208, 189, 255],
+                    '19': [221, 207, 188, 255],
+                    '20': [221, 207, 187, 255],
+                    '21': [220, 206, 185, 255],
```

Comment:
The legend looks bad because it overflows the screen, it needs to be limited vertically I guess.